Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1
+
+FROM python:2.7-slim-buster
```

Comment:
any reason for using python2.7? what version of dxpy is available for this? Probably should be using 3.5+

Reply:
my bad, copied from template, didn't even realise. Changed to python3.8-slim for the docker image pull